### PR TITLE
PT-3651: Add ability to consent/push/export genes and variants separately

### PR DESCRIPTION
**Pull request by veronikaslc**
Added new UIX genes extension that is generating UI only if gene-variants UIX is disabled

### CODE REVIEW

**Comment by veronikaslc** [comment]
@sashaandjic this PR is based on PT-3602 branch

**Comment by veronikaslc** [comment]
@sashaandjic ready to be tested

**Comment by veronikaslc** [comment]
should be neither, committed a fix for auto checking variants consent if genes consent is checked.

**Comment by a-gvo** [comment]
After both genes and variants are selected, i can still un-check just variants.

File: components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java [hunk 1]
```diff
@@ -294,9 +294,9 @@ public DataSection genesBody(Patient patient) throws Exception
 
     public void variantsSetup(Set<String> enabledFields) throws Exception
     {
-        String sectionName = "genes";
-        Set<String> present = this.enabledHeaderIdsBySection.get(sectionName);
-        if (present == null || present.isEmpty()) {
+        String sectionName = "variants";
```
Comment (sdumitriu) [inline components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java]: Excel export never includes variants with this change. `sectionName` must also be changed in the next two methods, `variantsHeader` and `variantsBody`,

**Pull request merged**